content/docs/tasks/security/authn-policy/index.md

```diff
@@ -617,7 +617,7 @@ EOF
 {{< /text >}}
 
 > If you already enable mutual TLS mesh-wide or namespace-wide, the host `httpbin.foo` is already covered by the other destination rule.
-Therefore, you do not need adding this destination rule. On the other hand, you still need to add the `mTLS` stanza to the authentication policy as the service-specific policy will override the mesh-wide (or namespace-wide) policy completely.
+Therefore, you do not need adding this destination rule. On the other hand, you still need to add the `mtls` stanza to the authentication policy as the service-specific policy will override the mesh-wide (or namespace-wide) policy completely.
```

Comment:
need to add this ...

Reply:
@frankbu Can you elaborate what needs to be added? Moreover, the circleci test failed with the following error, which does not seem to be related to this PR. I tried to issue /retest but circleci test does not re-run. Do you know how to resolve this circleci test failure?

- ./public/docs/setup/kubernetes/quick-start-gke-dm/index.html
  *  External link https://cloud.google.com/kubernetes-engine/versioning-and-upgrades#checking_available_versions failed: https://cloud.google.com/kubernetes-engine/versioning-and-upgrades exists, but the hash 'checking_available_versions' does not
htmlproofer 3.9.2 | Error:  HTML-Proofer found 1 failure!
LINTING FAILED

Reply:
I meant that you need to correct the grammar. Change "need adding this" to "need to add this" in the first sentence.

Don't worry about the circleci error, it's unrelated to your change.

Reply:
Done, thanks!